docs/using-tendermint.rst

```diff
@@ -93,6 +93,54 @@ Visit http://localhost:46657 in your browser to see the list of other
 endpoints. Some take no arguments (like ``/status``), while others
 specify the argument name and use ``_`` as a placeholder.
 
+Formatting
+~~~~~~~~~~
+
+The following nuances when sending/formatting transactions should
+be taken into account:
+
+With ``GET``:
+
+To send a UTF8 string byte array, quote the value of the tx pramater:
+
+::
+
+    curl http://localhost:46657/broadcast_tx_commit?tx="hello"
```

Comment:
I think we need single quotes `'` around the entire URL - otherwise bash will ignore the `"` .